jwst/flatfield/flat_field.py

```diff
@@ -288,6 +290,9 @@ def do_nirspec_flat_field(output_model, f_flat_model, s_flat_model, d_flat_model
                                    .format(type(output_model)))
             return nirspec_ifu(output_model, f_flat_model, s_flat_model, d_flat_model, dispaxis,
                                user_supplied_flat=user_supplied_flat, inverse=inverse)
+        else:
+            raise RuntimeError(f'No flat field algorithm exists for handling data {output_model}')
```

Comment:
Is `output_model` a datamodel here and, if so, how does that get printed into a log message?

Reply:
The repr of data models are that strings like "ModelType(size) filename". These happen all the time now in logging.